packages/handlersjs-http/lib/models/http-handler-context.ts

```diff
@@ -1,7 +1,9 @@
+import { Logger } from '@digita-ai/handlersjs-logging';
 import { HttpHandlerRequest } from './http-handler-request';
 import { HttpHandlerRoute } from './http-handler-route';
 
 export interface HttpHandlerContext {
   request: HttpHandlerRequest;
+  logger: Logger;
```

Comment:
Again, just a question: Adding this here doesn't result in any compatibility issues for anyone who is using handlersjs right? I presume that if they upgrade to the version that would include this change their project will run like usual.

Reply:
If the project uses the `NodeHttpRequestResponseHandler` there should be no issue. This handler will add a logger to the `HttpHandlerContext`, up to you if you use it I guess.

Reply:
Even if you don't use the `NodeHttpRequestResponseHandler`, the logger will just be undefined, right? I guess at that point if you try to call a log function on it it'll error, but if you leave everything as is it'll continue to work

Reply:
If someone designs their own handler that creates the `HttpHandlerContext` they will just have to put a logger into that object themselves.